src/index.js

```diff
@@ -0,0 +1,240 @@
+const state = {
+    temp: 48,
+    initTempColorClass: 'forty-degrees',
+    tempBox: document.getElementById('temp-box'), // NOT SURE IF THIS IS OK
+    landscapes_map: {
+        '80+': '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂',
+        '70+': '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+        '60+': '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃',
+        '50+': '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲',
+        '40+': '🌬🌨 ️⛄🌬🌨 ️⛄🌬🌨 ️⛄🌬🌨 ️',
+    },
+    sky_map: {
+        sunny: '  ☁️      ☀️ ☁️      ☁️ ☁️',
+        cloudy: '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️',
+        rainy: '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧',
+        snowy: '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨',
+    }
+}
+
+// does it make sense to use state this way?
```

Comment:
state is generally used to hold the current state of the app, so data that might change